cumulusci/tasks/apex/testrunner.py

```diff
@@ -512,6 +527,28 @@ def _run_task(self):
                 )
             )
 
+        if self.code_coverage_level:
+            if self.options.get("namespace") not in self.org_config.installed_packages:
+                self._check_code_coverage()
+            else:
+                self.logger.info("This is a managed org; not checking code coverage.")
+        else:
+            self.logger.info(
+                "No code coverage level specified; not checking code coverage."
+            )
+
+    def _check_code_coverage(self):
+        result = self.tooling.query("SELECT PercentCovered FROM ApexOrgWideCoverage")
+        coverage = result["records"][0]["PercentCovered"]
+        if coverage < self.code_coverage_level:
+            raise ApexTestException(
```

Comment:
Does using `ApexTestException` muddy the water too much? Perhaps this should be a new exception class.

Reply:
Using ApexTestException has the advantage of being in the list of exceptions that MetaCI will count as a failure rather than an error. Creating a new subclass of ApexTestException would also work.